src/devtools/client/inspector/rules/utils/utils.js

```diff
@@ -288,8 +288,21 @@ function hasCSSVariable(propertyValue, variableName) {
   return regex.test(propertyValue);
 }
 
+/**
+ * Returns filtered class names that are nullish or falsey, concatenated into a
+ * single class name.
+ *
+ * @param {Array} classNames
+ *        Array of class names (Array<string | null | false | undefined>).
+ * @return {String}
+ */
+const classes = (...classNames) => {
```

Comment:
We usually use the `classnames` package, which is also more versatile (it can not only take an array but also an object where the keys are classnames and only those with a truthy value are used - that allows optional classnames to be specified in a slightly more readable way)